applications/power/power_service/power.c

```diff
@@ -204,10 +276,19 @@ static void power_check_battery_level_change(Power* power) {
 int32_t power_srv(void* p) {
     UNUSED(p);
     Power* power = power_alloc();
+    if(!LOAD_POWER_SETTINGS(&power->shutdown_idle_delay_ms)) {
+        memset(&power->shutdown_idle_delay_ms, 0, sizeof(uint32_t));
```

Comment:
why not just = 0?

Reply:
That's a leftover of an initial idea of having a structure of options but then I found out I only needed one. I'll fix it asap.